packages/lib/bookingSuccessRedirect.ts

```diff
@@ -36,9 +39,39 @@ export const getBookingRedirectExtraParams = (booking: SuccessRedirectBookingTyp
     "location",
   ];
 
-  return (Object.keys(booking) as BookingResponseKey[])
+  const basicParams = (Object.keys(booking) as BookingResponseKey[])
     .filter((key) => redirectQueryParamKeys.includes(key))
     .reduce((obj, key) => ({ ...obj, [key]: booking[key] }), {});
+
+  const serializeResponseValue = (value: any): string => {
+    if (Array.isArray(value)) {
+      return value.join(",");
+    } else if (typeof value === "object" && value !== null) {
+      return encodeURIComponent(JSON.stringify(value));
+    } else {
+      return String(value);
+    }
+  };
+
+  const responseParams = booking.responses
+    ? Object.keys(booking.responses as Record<string, any>).reduce((obj, key) => {
+        obj[`responses.${key}`] = serializeResponseValue((booking.responses as Record<string, any>)[key]);
+        return obj;
+      }, {} as Record<string, any>)
+    : {};
+
+  const meetingLink = (booking.references as { meetingUrl?: string }[] | undefined)?.find(
+    (ref: { meetingUrl?: string }) => ref.meetingUrl
+  )?.meetingUrl;
+
+  if (meetingLink) {
+    responseParams["responses.location"] = meetingLink;
+  }
```

Comment:
We are passing 'location' as the default in the parameters, so you need to filter it out from the response. Otherwise, it will display both 'location:' and 'response.location:'. If 'location' is present, overwrite the video call URL with it

Reply:
What's the issue if it shows both `location` and `response.location` because response is something we are getting from the API response.

Reply:
Also can we get the PR approved and merged, I think it's functional and we are drifting off for a while now.